.github/workflows/atlas-tests.yml

```diff
@@ -0,0 +1,134 @@
+## Github workflow to run atlas tests
+
+name: Tests::Atlas
+
+on:
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  BITCOIND_TEST: 1
+  RUST_BACKTRACE: full
+  SEGMENT_DOWNLOAD_TIMEOUT_MINS: 15
+
+concurrency:
+  group: atlas-tests-${{ github.workflow }}-${{ github.ref }}
+  # Only cancel in progress if this is for a PR
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  # atlas integration tests with code coverage
+  atlas-tests:
+    name: Atlas Test
+    runs-on: ubuntu-latest
+    ## run a maximum of 2 concurrent tests from the test matrix
+    strategy:
+      fail-fast: false
+      max-parallel: 2
```

Comment:
Clarifying - is this "max-parallel" required when the matrix only has two options?

Reply:
it is - without that directive, the tests will run serially essentially doubling the test time (give or take)